### ISSUE TITLE
Can not compile on latest build due to core library "format!" conflict

### ISSUE THREAD

**Issue by xentrick** [librespot-org/librespot#370]
Seems that ncspot will not compile on the latest nightly build.
```
/.cargo/git/checkouts/librespot-06fda9f186b35c32/1ce0706/metadata/src/lib.rs:62:19
   |
62 |         let uri = format!("{}/{}", Self::base_url(), id.to_base16());
   |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ could not find `format_args` in `core`
   |
   = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)
```
This seems related to the following extern as `core`. The compiler is attempting to locate `format!` within `librespot_core` instead of `core::fmt`
```
extern crate librespot_core as core;
```

**Comment by HEnquist** [comment]
I have a PR with the fix for this issue:
https://github.com/librespot-org/librespot/pull/366

**Comment by xentrick** [comment]
@HEnquist Thanks, I forked and did the same fix to get it running for myself. Hopefully the changes get merged soon.

**Comment by ashthespy** [comment]
Fixed by #371
